pipeline/scripts/2-DIMS.R

```diff
@@ -69,6 +69,13 @@ negRes=NULL
 
 x <- suppressMessages(xcmsRaw(filepath))
 
+# New: generate TIC plots. Prepare txt files with data for plots
+TIC_intensity_persample <- cbind(round(x@scantime, 2), x@tic)
+# for mzR package, use:
+# TIC_intensity_persample <- cbind(round(hdr$retentionTime, 2), hdr$totIonCurrent)
```

Comment:
Seems like a redundant comment?

Reply:
This comment is for the stitching pipeline, not appropriate here. Removed.